packages/terra-table/src/proptypes/columnShape.js

```diff
@@ -42,6 +42,18 @@ const columnShape = PropTypes.shape({
    * One of `ascending`, `descending`.
    */
   sortIndicator: PropTypes.oneOf(Object.values(SortIndicators)),
+
+  /**
+   * @private
+   * The color to be used for highlighting a column.
+   */
+  columnHighlightColor: PropTypes.oneOf(['orange', 'green']),
```

Comment:
Use the constants here

Reply:
You should be using your enumeration.  Also, why is this a private property?  We want consumers to be able to set this, correct?

Reply:
Changed to enum in https://github.com/cerner/terra-framework/pull/1985/commits/8a2aac51a800bf755283f26bbdb29313b3703c04
`columnHighlightColor` and `columnHighlightDescription` are only exposed through the column object from flowsheet.
 Column table under flowsheet lists these attributes.